src/main/webapp/js/components/entities/users/user-edit.jsx

```diff
@@ -0,0 +1,152 @@
+        }),
+      ),
+    };
+    this.props.updateEntity(entityType, entityValue, () => {
+      Alert.success('User has been updated!', {
+        timeout: ALERT_TIMEOUT,
+      });
+      this.props.history.push('/viewEntity/user');
+    });
+  };
+
+  validate = values => validate(this.props.fieldConfig)(values);
+
+  openConfirmModal = () => {
+    this.setState({ openConfirmModal: true });
+  };
+
+  hideConfirmModal = () => {
+    this.setState({ openConfirmModal: false });
+  };
+
+  deleteEntity = () => {
+    const { entityType, entityId } = this.props;
+    this.props.deleteEntity(entityType, entityId, () => {
+      Alert.success('User has been deleted!', {
+        timeout: ALERT_TIMEOUT,
+      });
+      this.props.history.push('/viewEntity/user');
+    });
+  };
+
+  isAuthorizated = (requiredPermissions) => {
+    const { permissions } = this.props;
+    if (_.isEmpty(requiredPermissions)) {
+      return true;
+    }
+    return _.every(requiredPermissions, permission => _.includes(permissions, permission));
+  };
+
+  render() {
+    const { openConfirmModal } = this.state;
+    const {
+      entityToEdit, isOnline, fieldConfig,
+    } = this.props;
+    return (
+      <div className="container">
+        <div>
+          <h1>
+            {'Edit User'}
+          </h1>
+        </div>
+        <div>
+          <Form
+            onSubmit={this.onSubmit}
+            validate={this.validate}
+            initialValues={entityToEdit}
+            render={({ handleSubmit }) => (
+              <form onSubmit={handleSubmit} className="modal-fields">
+                <div>
+                  {_.map(fieldConfig, elem => renderFormField({ ...elem, name: (elem.base ? elem.name : `extraFields.${elem.name}.value`), editable: this.isAuthorizated([ADMIN]) && elem.name === 'roleIds' ? true : elem.editable }))}
+                </div>
+                <div className="text-center">
+                  <button
+                    type="submit"
+                    className="btn btn-success btn-lg margin-top-sm padding-left-lg padding-right-lg margin-right-sm"
+                    disabled={!isOnline}
+                  >
+                    Update
+                  </button>
+                  <button
+                    type="button"
+                    className="btn btn-danger btn-lg margin-top-sm padding-left-lg padding-right-lg"
+                    onClick={this.openConfirmModal}
+                    disabled={!isOnline}
+                  >
+                    Delete
+                  </button>
+                </div>
+              </form>
+            )}
+          />
+        </div>
+        <ConfirmModal
+          showModal={openConfirmModal}
+          modalText="Are you sure to delete this?"
+          onConfirm={this.deleteEntity}
+          onHide={this.hideConfirmModal}
+        />
+      </div>
+    );
+  }
+}
+
+const mapStateToProps = (state, props) => ({
+  entityToEdit: getEntityMemberById(state, props),
```

Comment:
you should change the EntityEdit component and move this to EditEntityPage, so the EntityEdit will get the entityToEdit as a parameter